Library/Formula/cassandra.rb

```diff
@@ -28,23 +64,26 @@ def install
 
     rm Dir["bin/*.bat", "bin/*.ps1"]
 
+    rm_rf HOMEBREW_PREFIX/"etc/cassandra" # Error: File exists - /usr/local/etc/cassandra/triggers
```

Comment:
I'm happy to pull without this line and figure it out later if this doesn't affect upgrades.